quinn-proto/src/endpoint.rs

```diff
@@ -286,7 +286,9 @@ impl Endpoint {
                     None
                 }
             };
-        } else if first_decode.has_long_header() {
+        }
```

Comment:
While they are technically equivalent, I feel like the `else` here (in addition to being slightly more compact) is useful because it makes clear that there's no follow-through from the previous branch without relying on context (that the previous branch always diverges).

Reply:
Yeah I can see it. Removed commit.